ecs-init/cache/cache.go

```diff
@@ -37,6 +37,21 @@ const (
 	orwPerm = 0700
 )
 
+type CacheStatus uint8
+
+const (
+	// StatusUncached indicates that there is not an already downloaded
+	// and cached agent that is suitable for loading.
+	StatusUncached CacheStatus = iota
```

Comment:
If you're serializing these constants (which you are), please don't use `iota`.  It should be explicit when these numbers change meaning rather than implicit.

Reply:
Sounds good.